frontend/src/pages/following-page/tabs/offline-videos/offline-videos-tab.tsx

```diff
@@ -6,7 +6,7 @@ import { useAppDispatch, useAppSelector } from 'hooks/hooks';
 import React, { FC, useEffect } from 'react';
 import { shallowEqual } from 'react-redux';
 import { loadOfflineVideos } from 'store/following-page/actions';
-import { NoVideosYet } from '../common/no-videos-yet/no-videos-yet';
+import { NoVideosYet } from '../../../../components/common/no-videos-yet/no-videos-yet';
```

Comment:
your import should be replaced by 'components/common/...'.
and in other similar cases too.